### ISSUE TITLE
[Troubleshooting]: vanilla homebrew installation with opnJDK 18 fails to start

### ISSUE THREAD

**Issue by mischwartz23** [eXist-db/exist#4297]
:balloon: First off, thank you for contributing to eXist-db's documentation. :balloon:

**Describe the problem**
> Troubleshooting: Running eXist-db with openjdk on a Mac. Vanilla installation (via brew) leads to a pop-up error that Java 8 must be installed.

### Please provide the following
*   exist-db version: `6.0.1`
*   documentation version: `6.0.1`

TL;DR: I found that **modification of the Info.plist** and **linking the openjdk to the PlugIns** resolved the problem.

# Modification of the Info.plist

This file is located at `/Application/exist-db.app/Contents/Info.plist`

The modification is to add two lines to the file, just above the JVMOptions entry:

```
<key>JVMRuntime</key>
<string>openjdk</string>
```

This provides the identifier for the JVMRuntime, and will be the link source name in the PlugIns subdirectory.

# Linking the openjdk to the PlugIns subdirectory

Change directories to `/Applications/exist-db.app/Contents/PlugIns`

Find the relatively stable value for your openjdk installation. For me, this is in my `/Library/Java/JavaVirtualMachines/openjdk.jdk` link. It may be different for your installation (e.g., `/usr/local/opt/openjdk/libexec/openjdk.jdk`)

Assuming the directory for your jdk is as above, execute the command

`ln -s /Library/Java/JavaVirtualMachines/openjdk.jdk openjdk`

You should be able to launch the application normally at this point.

Reference: Original idea was found in [stackexchange](https://apple.stackexchange.com/questions/426713/how-to-help-the-jedit-application-find-java)

**Comment by joewiz** [comment]
Indeed, Homebrew [specifies](https://github.com/Homebrew/homebrew-cask/blob/master/Casks/exist-db.rb#L16) Java **8**:

> exist-db requires Java 8. You can install it with:
>
>  `brew install --cask homebrew/cask-versions/temurin8`

See also https://github.com/eXist-db/exist/issues/3375. I assume Java 18 would suffer from the same issue.

**Comment by dizzzz** [comment]
I  think/know there are a lot of (potential) issues with Java18, as discussed, we need to change code in order to be able run eXist-db on newer versions of Java.

So no warranty, and IMO yet no support for other Runtimes that Java8 and Java11

**Comment by mischwartz23** [comment]
Thanks for your comments. I appreciate and respect your perspectives.
I would point out that the provided directions would/will work with any installation of Java, and leave it at that.
Personally, I intend to install one (or maybe two) versions of Java and rely on command line flags if I require a different standard.

**Comment by duncdrum** [comment]
I propose to leave this issue open, so that others might find it. But to flag it as a no-fix. Once exist supports java versions beyond 12 we can revisit our official recommendation and the installer scripts. 
Thank you @mischwartz23 for providing the above steps
